### DISCUSSION TITLE
Can I validate collection/bundle label without performing referential integrity?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/validate#512] [Q&A] [answered]
Per https://github.com/NASA-PDS/operations/issues/239#issuecomment-1145394761, @smclaughlin7 asks:

Can you confirm that the validation tool can be run on just the bundle product file or a collection product (its collection label and inventory file)? I recall being able to do this years ago, just to make sure the bundle file or a collection product is OK, without having the tool drill down and validate all the children products. I assume validate can still be run this way? Thanks!! 

@beatricemueller

**Comment by jordanpadams** [comment, marked as answer]
@smclaughlin7 yes.

```
validate --rule pds4.label --target <collection/bundle_label>
```

I think this will also just work like this, but not sure:

```
validate --target <collection/bundle_label>
```

**Reply by smclaughlin7** [reply]
@jordanpadams Thanks!
